printers/printerTree.php

```diff
@@ -197,13 +215,9 @@ private function _printSubTree($tree, $level) {
  * Represent a namespace and its inner content
  */
 class NspagesTreeNsNode implements ArrayAccess {
-    /**
-     * The list of pages directly in the namespace (does not include pages in subnamespaces)
-     */
-    public $pages = array();
 
     /**
-     * The list of subnamespaces at level n+1 (does not include their own subnamespaces)
+     * The list of pages and subnamespaces at level n+1 (does not include their own subnamespaces)
```

Comment:
Just to make sure I understand this part correctly: IIUC we merge `$pages` and `$children` in order to have the namespaces leaves correctly sorted among pages leaves.
Is my understanding correct?

Reply:
Yes, I merged they, page can have children, that means that maybe there is a page for NS. Tool at my structure, you see directory proxmox and there is file proxmox.txt. Proxmox page has childrens.
![obraz](https://github.com/user-attachments/assets/0bb1f2d9-871e-46ec-ade4-77ee30f2c723)